public/app/features/dashboard-scene/scene/DashboardSceneRenderer.tsx

```diff
@@ -55,6 +57,31 @@ export function DashboardSceneRenderer({ model }: SceneComponentProps<DashboardS
     <DashboardEmpty dashboard={model} canCreate={!!model.state.meta.canEdit} key="dashboard-empty-state" />
   );
 
+  const showAngularDeprecationNotice = () => {
```

Comment:
would turn this into a plain react component named `DashboardAngularDeprecationBanner` or something 

alternatively rename it to module level function named `renderAngularDeprecationBanner(model) (outside this react components render)

nitpick: but function name showXX does not feel like react render function but more like a state action